chain_harvester/chain.py

```
@@ -107,43 +119,75 @@ def get_latest_block(self, true_latest=False):
+
+            log.error(
+                "ABI for %s was fetched from 3rd party service. Add it to abis folder!",
+                contract_address,
+            )
+            abi = self._fetch_abi_from_chain(contract_address)
+
+            with open(file_path, "w") as f:
+                json.dump(abi, f)
+            self._abis[contract_address] = abi
+
+    def load_abi(self, contract_address, refetch_on_block=None, **kwargs):
```

Ruff (0.13.1)

Unused method argument: `kwargs`

(ARG002)